app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -61,14 +61,20 @@ def allowed_customization_templates(options = {})
     end
   end
 
-  def allowed_hosts_obj(options = {})
-    return [] if (src = resources_for_ui).blank?
-    return [] unless src[:vm]
+  def allowed_datacenters(_options = {})
+    #byebug_term
```

Comment:
This is branch will never be up for merge and will be deleted very soon, I just shared it with someone to help me figure something out with the tests. (Is there any kind of tag except "WIP" I should have added?)